src/QtDBMigration.cpp

```diff
@@ -36,20 +37,27 @@ class QtDBMigrationPrivate
     bool migrate();
     bool migrate(int version);
 
+
 private:
     bool createMetaTable();
 
     QtDBMigration *m_q;
     AbstractConfig *m_config;
+    AbstractConfig *m_folder_config;
 };
 
 QtDBMigrationPrivate::QtDBMigrationPrivate(QtDBMigration *q) :
     m_q(q) ,
-    m_config(new JsonConfig(q->m_configPath))
+    m_config(new JsonConfig(q->m_configPath)),
+    m_folder_config(new FolderConfig(q->m_configPath))
+
 {
     // Load config
     if (!m_config->load()) {
-        qWarning() << "Fail to load migration config";
+        qWarning() << "Fail to load migration json config";
```

Comment:
This is unneeded.

Reply:
Reverted back to a more general message.